src/test/scala/nl.knaw.dans.easy.validatebag/rules/structural/StructuralRulesSpec.scala

```diff
@@ -126,13 +126,21 @@ class StructuralRulesSpec extends TestSupportFixture {
     )
   }
 
-  it should "fail if a payload file has invalid characters" in {
+  it should "fail if a payload file name has invalid characters" in {
```

Comment:
as in https://github.com/DANS-KNAW/dans-bagit-profile/pull/20
```suggestion
  it should "fail if a payload file path has invalid characters" in {
```

Reply:
I combined the two tests